src/deploy/missions/contracts/z-token/z-token.ts

```diff
@@ -86,35 +86,36 @@ IZNSContracts
   }
 
   getArtifact () {
-    return meowArtifact;
+    return zArtifact;
```

Comment:
you need to change this. we did it before because the other repo was bugged.
This repo should work fine, so you can remove this method completely. HardhatDeployer should be able to get the artifact where needed in it's default way.